pyccel/codegen/printing/ccode.py

```diff
@@ -1005,7 +1005,15 @@ def _print_Import(self, expr):
                               f'#define i_key {container_key}',
                               f'#include "{stc_name + "/" + container[0]}.h"',
                               '#endif\n'))
-
+        elif source.startswith('Set_pop'):
+            _ , i_type, i_key = source.split('/')
+            self.add_import(Import('STC_Extensions', Module('STC_Extensions', (), ())))
+            return '\n'.join((
+                   f'#ifndef TOOLS_SET_{str(i_key).upper()}\n'
+                   f'#define TOOLS_SET_{str(i_key).upper()}\n'           
+                   f'#define i_type {i_type}',
+                   f'#define i_key {i_key}\n',
+                   f'#include "STC_Extensions/Set_extensions.h"\n#endif\n'))
```

Comment:
STC extensions is not really a library so I don't think it is necessary to include it via the library folder

```suggestion
                   f'#include "Set_extensions.h"\n#endif\n'))
```

Reply:
It's a good option if we have the `List_extensions.h` file in the future.

Reply:
What is a good option?
For me a library is something that can be downloaded on its own and used. So STC is a library, Pyccel is a library. If we put STC extensions in its own repo then it could be a library, but as is it is just a collection of functions provided by Pyccel to work round STC's limitations

Reply:
Fixed : 8000948